zerver/lib/message.py

```diff
@@ -320,6 +322,7 @@ def render_markdown(message, content, realm=None, realm_alert_words=None, messag
             if user_id in message_user_ids:
                 possible_words.update(set(words))
 
+    slash_commands = set(get_slash_commands_by_realm(realm, is_active=True))
```

Comment:
It looks like this will be called very often.  What's the performance of `get_slash_commands_by_realm`, and how will that affect the overall performance here?